packages/react-components/react-link/src/components/Link/useLinkStyles.styles.ts

```diff
@@ -17,6 +17,9 @@ const useStyles = makeStyles({
   }),
   // Common styles.
   root: {
+    ':focus-visible': {
+      outlineStyle: 'none',
```

Comment:
Isn't this achieving the same thing as line 16 above? (Judging from the code in  `createCustomFocusIndicatorStyle`)

Looking at `createFocusOutlineStyle`, we might be missing doing `outlineStyle: 'none'` for `focus` instead of `focus-visible` which we do account if we pass the `outlineStyle` to `createCustomFocusIndicatorStyle`.